fgcm/fgcmDeltaAper.py

```diff
@@ -522,10 +522,10 @@ def _starWorker(self, goodStarsAndObs):
 
         np.add.at(objDeltaAperMeanTemp,
                   (obsObjIDIndex[goodObs], obsBandIndex[goodObs]),
-                  (obsDeltaAper[goodObs] - self.fgcmPars.compMedDeltaAper[obsExpIndex[goodObs]])/obsMagErr2GO)
+                  ((obsDeltaAper[goodObs] - self.fgcmPars.compMedDeltaAper[obsExpIndex[goodObs]])/obsMagErr2GO).astype(objDeltaAperMeanTemp.dtype))
```

Comment:
Formatting?

Reply:
🤷‍♂️

Reply:
Oh, it may have just looked funny on my screen.  Ignore!